src/bitcoin_node_cli.hpp

```diff
@@ -3,6 +3,20 @@
 
 #include <libs0.hpp>
 #include <boost/asio.hpp>
+#include <curl/curl.h>
+#include <mutex>
+
+class c_curl_ptr final {
+	public:
+		c_curl_ptr();
+		~c_curl_ptr();
+		CURL *get_raw_ptr() const;
+		static std::once_flag s_curl_init_flag;
```

Comment:
Should not be public. Better make it private, but make a static public function init_this_class...

BUT instead, we do not need this at all, remove this flag - if it is used only in main() now.

Reply:
seems done (as of 7fa89ee)